pvlib/temperature.py

```diff
@@ -443,6 +444,50 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
     return temp_air + temp_difference
 
 
+def ross(poa_global, temp_air, noct):
+    r'''
+    Calculate cell temperature using the Ross model.
+
+    The Ross model [1]_ assumes that cell temperature is proportional to the
+    plane of array irradiance, and ignores the effects of wind.
+
+    Parameters
+    ----------
+    poa_global : numeric
+        Total incident irradiance [W/m^2].
+
+    temp_air : numeric
+        Ambient dry bulb temperature [C].
+
+    noct : numeric
+        Nominal operating cell temperature, determiend at conditions of
+        800 W/m^2 irradiance, 20C ambient air temperature and 1 m/s wind.
+
+    Returns
+    -------
+    numeric, values in degrees Celsius
+
+    Notes
+    -----
+    The Ross model for cell temperature :math:`T_{C}` is given in [1]_ as
+
+    .. math::
+       :label: pvsyst
+
+        T_{C} = T_{a} + \frac{NOCT - 20}{80} S
+
+    where :math:`S` is the plane of array irradiance in mW/{cm}^2.
```

Comment:
Are these braces intended?  Or maybe the unit should be wrapped in a `:math:` tag?